tests/plugins/security/api/securityconfig.yaml

```diff
@@ -0,0 +1,58 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+# ADMIN-CERT only (except GET). These tests require explicit rest api admin privileges.
+
+description: Test securityconfig endpoints.
+
+chapters:
+  - synopsis: Update a security config.
+    path: /_plugins/_security/api/securityconfig/config
+    method: PUT
+    request:
+      payload:
+        dynamic: 
+          filtered_alias_mode: warn
+          disable_rest_auth: false
+          disable_intertransport_auth: false
+          respect_request_indices_options: false
+          opensearch-dashboards: 
+            multitenancy_enabled: true
+            server_username: kibanaserver
+            index: .opensearch-dashboards
+          http: 
+            anonymous_auth_enabled: false
+          authc: 
+            basic_internal_auth_domain: 
+              http_enabled: true
+              transport_enabled: true
+              order: 0
+              http_authenticator: 
+                challenge: true
+                type: basic
+                config: {}
+              authentication_backend: 
+                type: intern
+                config: {}
+              description: Authenticate via HTTP Basic against internal users database
+          auth_failure_listeners: {}
+          do_not_fail_on_forbidden: false
+          multi_rolespan_enabled: true
+          hosts_resolver_mode: ip-only
+          do_not_fail_on_forbidden_empty: false
+    response:
+      status: 403
+  - synopsis: Get a security config.
+    path: /_plugins/_security/api/securityconfig
+    method: GET
+    response:
+      status: 200
+  - synopsis: Patch a security config.
+    path: /_plugins/_security/api/securityconfig
+    method: PATCH
+    request:
+      payload:
+        - op: replace
+          path: /config/dynamic/authc/basic_internal_auth_domain/transport_enabled
+          value: 'true'
```

Comment:
This is not a bool?

Reply:
accepts either or. will change it to bool